packages/client/src/components/Header/Header.tsx

```diff
@@ -361,13 +361,13 @@ class HeaderComp extends React.Component<IFullProps, IState> {
     const role = this.props.userDetails && this.props.userDetails.role
     const location = this.props.history.location.pathname
     if (
-      (FIELD_AGENT_ROLES.includes(role as string) && location.includes(HOME)) ||
+      (FIELD_AGENT_ROLES.includes(role as string) && HOME.includes(location)) ||
       (NATL_ADMIN_ROLES.includes(role as string) &&
-        location.includes(PERFORMANCE_HOME)) ||
+        PERFORMANCE_HOME.includes(location)) ||
       (SYS_ADMIN_ROLES.includes(role as string) &&
-        location.includes(PERFORMANCE_HOME)) ||
+        PERFORMANCE_HOME.includes(location)) ||
       (REGISTRAR_ROLES.includes(role as string) &&
-        location.includes(REGISTRAR_HOME))
+        REGISTRAR_HOME.includes(location))
```

Comment:
These changes will cause this bug to reappear https://github.com/opencrvs/opencrvs-core/issues/4203 so we should not do this

Reply:
okay